File: server/App.Domain/Entities/Perfil.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace App.Domain.Entities
{
    [Table("Perfil")]
    public class Perfil : EntityBase
    {
        public string Nome { get; set; }

        public virtual List<Usuario> Usuario { get; set; }

        public virtual List<PerfilFuncionalidade> PerfilFuncionalidade { get; set; }
    }
}

File: client/src/app/shared/models/perfil.ts
import { PerfilFuncionalidade } from './perfilFuncionalidade';

export class Perfil {
  id: string;
  nome: string;
  dataCadastro: Date;
  dataAtualizacao: Date;
  excluido: boolean;
  perfilFuncionalidade: PerfilFuncionalidade[];
}
